impress-org/givewp
# Fix: remove excess form builder padding on classic themes

## Description

I finally figured out why some folks were seeing weird extra padding in the form builder and others weren't. It had to do with which theme was loaded. Apparently, classic themes — that is, not block themes — load an additional CSS file in the builder: `wp-includes/css/dist/edit-post/classic.css`. This was causing extra padding to occur on our blocks.

I thought about dequeuing the style altogether, and even tried, but decided to leave it alone in case it's being relied on in some way we're not aware of. So instead, I just added a small bit of CSS which attempts to undo the specific styles causing issues.

## Affects

Blocks in the form builder

## Visuals

### Before

<img width="749" alt="image" src="https://github.com/impress-org/givewp/assets/2024145/f7b4ee9f-69d6-4338-9a08-05d102bfd847">

### After

<img width="751" alt="image" src="https://github.com/impress-org/givewp/assets/2024145/a0291c81-d77b-45ac-92e5-b798adb62489">


## Testing Instructions

1. Activate the Twenty Twenty-One theme
2. View the builder
3. Recoil in horror
4. Update to this PR
5. Check builder again
6. Nod calmly

## Pre-review Checklist
